frontend/src/scenes/performance/webPerformanceLogic.ts

```diff
@@ -330,7 +331,11 @@ export const webPerformanceLogic = kea<webPerformanceLogicType<EventPerformanceD
     loaders: ({ values }) => ({
         pageViewEvents: {
             loadEvents: async () => {
-                const combinedProperties = [...(eventApiProps.properties || []), ...values.properties]
+                const flattenedPropertyGroup =
+                    eventApiProps.properties && isPropertyGroup(eventApiProps.properties)
```

Comment:
don't think you need to null check `eventApiProps.properties` before isPropertyGroup because the function itself will handle the case if it's null. This is done in a few other areas where isPropertyGroup is called too